internal/services/pam/testdata/fixtures/invalid_brokers/no_dbus_name

```diff
@@ -0,0 +1,6 @@
+name = Broker
+brand_icon = some_icon.png
```

Comment:
I think it’s a little bit too granular here. I would just have one or two invalid with some missing config. The granularity on what is missing resulting in the broker being ignored belongs to internal/brokers/ tests. Do we have tests for those here?